modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js

```diff
@@ -198,7 +218,7 @@ export default withRouter(
 				/>
 
 				<div className="c-mt-5">
-					{!loading && (
+					{!loading && !error && (
```

Comment:
😂 order I guess?

Reply:
😂  Exactly. I've learned that the setLoading always has to be called at the end

Reply:
![](https://media.giphy.com/media/tJMVcTfzDdL1pOGxlk/giphy.gif)